src/sentry/api/endpoints/release_deploys.py

```diff
@@ -90,12 +94,37 @@ def post(self, request: Request, organization, version) -> Response:
                                       the deploy ended. If not provided, the
                                       current time is used.
         """
+        logging_info = {
+            "org_slug": organization.slug,
+            "org_id": organization.id,
+            "version": version,
+        }
+
         try:
             release = Release.objects.get(version=version, organization=organization)
         except Release.DoesNotExist:
+            logger.info(
+                "create_release_deploy.release_not_found",
+                extra=logging_info,
+            )
             raise ResourceDoesNotExist
 
         if not self.has_release_permission(request, organization, release):
+            # Logic here copied from `has_release_permission` (lightly edited for results to be more
+            # human-readable)
+            auth = None
+            if getattr(request, "user", None) and request.user.id:
+                auth = f"user.id: {request.user.id}"
+            elif getattr(request, "auth", None) and getattr(request.auth, "id", None):
+                auth = f"auth.id: {request.auth.id}"  # type: ignore
+            elif getattr(request, "auth", None) and getattr(request.auth, "entity_id", None):
+                auth = f"auth.entity_id: {request.auth.entity_id}"  # type: ignore
+            if auth is not None:
```

Comment:
nit: shouldn't this be called `actor_id` instead of `auth`? I think that's what this logic is capturing if I"m reading the original logic correctly

Reply:
`auth` is just a human-readable label for it. Yes, when this is used for anything other than logging, it's `actor_id`, but in that case the values are also slightly different (no differentiation between `auth.id` and `auth.entity_id`, for example).

You're right, maybe for perfect debugging, I should do the exact calculation and separately note the auth type, but since (as you say) this is temporary, I'm going to merge this as is.